src/pymatgen/io/packmol.py

```diff
@@ -1,20 +1,17 @@
 """
-This module provides a pymatgen I/O interface to packmol.
+This module provides a pymatgen I/O interface to PACKMOL.
 
-This adopts the minimal core I/O interface (see pymatgen/io/core).
-In this case, only a two classes are used. PackmolSet(InputSet) is the container
-class that provides a run() method for running packmol locally.
+- PackmolSet provides a "run" method to run PACKMOL locally.
 
-PackmolBoxGen(InputGenerator) provides a recipe for packing molecules into a
-box, which returns a PackmolSet object.
+- PackmolBoxGen provides "get_input_set" for packing molecules into a box,
+which returns a PackmolSet object.
 
-For the run() method to work, you need to install the packmol package
-See http://m3g.iqm.unicamp.br/packmol or
-http://leandro.iqm.unicamp.br/m3g/packmol/home.shtml
```

Comment:
The second URL doesn't seem to work (at least on my side). 

Also removed some implementation details from the docstring (from explain the parent class `PackmolBoxGen(InputGenerator)` and "This adopts the minimal core I/O interface (see pymatgen/io/core).").